pages/schema.md

```diff
@@ -196,25 +208,33 @@ historicalCostSavings |  String, 5 - 10000              | Yes      |
 			"fy19Achieved": 0,
 			"fy20Planned": 20,
 			"fy20Achieved": 0
+			"fy21Planned": 21,
```

Comment:
For all of these FY 21 JSON additions, you're missing a comma to end the line above (`fy20Achieved`).  You might also consider adding fy22 at this point to save some heartache later, given that this is already in law.

Reply:
thanks for the review, hopefully caught all the issues this time